test/engine-tap/datetime.test.lua

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env tarantool
+
+package.path = "lib/?.lua;"..package.path
```

Comment:
Why do you need this line?

Reply:
To make it self-running, outside of `test-run.py`

Reply:
I dropped this line and still was able to run it without test-run. But who cares!